tests/test_runs/test_run_functions.py

```diff
@@ -1121,7 +1121,7 @@ def test_get_run(self):
 
     def _check_run(self, run):
         self.assertIsInstance(run, dict)
-        self.assertEqual(len(run), 7)
+        assert len(run) == 7, str(run)
```

Comment:
can you please explain in comments why 7? is not clear from the (direct) context

Reply:
That's the amount of entries per run returned by the API: https://openml.org/api/v1/xml/run/list/flow/1154

Reply:
I just pushed a commit explaining this. I will merge once the tests are through.

Reply:
good. I am not sure if we should really check this, as this is flexible for extension. But in those cases we can adjust the unit tests as well